src/EditorFeatures/CSharp/AutomaticCompletion/AutomaticLineEnderCommandHandler.cs

```diff
@@ -589,16 +589,35 @@ private static int GetOpenBraceSpanEnd(SyntaxNode root)
         }
 
         private static int GetBraceInsertionPosition(SyntaxNode node)
-            => node switch
+        {
+            if (node is SwitchStatementSyntax switchStatementNode)
```

Comment:
Our syntax tree always prefers SwitchStatement,
even just type a 'switch' it still thinks this is a switch statement.
So here I suppose if the switch statement doesn't have '( )', then it is thought as SwitchExpression

Reply:
I find this interesting given that the expression/identifier for the switch comes before the switch keyword for switch expressions.